configure.in

```diff
@@ -156,7 +157,6 @@ Use --without-libcurl to disable anonymous statistics collection.])])
 If you have libcurl already installed, see config.log for details on the
 failure.  It is possible the compiler isn't looking in the proper directory.
 Use --without-libcurl to disable libcurl support.])])
-  CITUS_CFLAGS="$CITUS_CFLAGS -DHAVE_LIBCURL"
```

Comment:
The `AC_DEFINE` just adds a constant to `citus_config.h` and since we cannot include it we don't see the `HAVE_LIBCURL`. That is why I added this line. Sorry for missing this in the review of #1701 .

Reply:
aaah ... I see you have this in `CITUS_VERSION` too. I'll look into this. Nevermind the previous comment.

Reply:
I think what was missing was just a include to `citus_version.h`. I'll add that.

Reply:
Yeah, if I add a `AC_DEFINE`, I'll usually run `autoheader` to have it generate the section in `citus_config.h`, then I'll copy-paste that section into `citus_version.h`.